keps/sig-node/2371-cri-pod-container-stats/kep.yaml

```diff
@@ -16,13 +16,12 @@ approvers:
 prr-approvers:
   - "@ehashman"
 creation-date: 2021-01-27
-last-updated: 2022-01-25
+last-updated: 2022-06-13
 status: implementable
-stage: beta
-latest-milestone: "v1.24"
+stage: alpha
```

Comment:
This is targeting `beta` in v1.25, right?  If so, `stage`, `latest-milestone` and `milestone.beta` needs to reflect that.

Reply:
actually no, this is an update to the alpha stage. It was originally targeting beta but I reconsidered